linode-launch.py

```diff
@@ -48,85 +49,136 @@ def releasing(semaphore):
 create_semaphore = BoundedSemaphore(15)
 config_semaphore = BoundedSemaphore(8)
 def do_create(client, running, datacenter, plans, distribution, kernel, machine, i):
-    plan = filter(lambda p: p[u'LABEL'].lower().find(str(machine['plan']).lower()) >= 0, plans)[0]
+    
+    for p in plans:
+        str_plan_label = p.label
```

Comment:
Can you add a comment with an example plan so the reader of the code knows what's going on?